implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -187,12 +175,14 @@ pub(crate) async fn background_mode(
     ctx: Context,
     (opts, cmd): (CommandGlobalOpts, CreateCommand),
 ) -> miette::Result<()> {
-    let node_name = &parse_node_name(&cmd.node_name)?;
```

Comment:
If we remove this function we lose the ability to accept both `/node/n1` and `n1` as valid node names. Is that something we want to do? I would vote for just accepting `n1`.

Reply:
This is now done inside the `Background::create` function, to avoid doing it everywhere. I think this is wrong though! We should rather have a `NodeName` struct and an associated parser which is able to parse from either `/node/n1` or `n1`.